data/attributes.yml

```diff
@@ -30209,6 +30209,46 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3413
+  name: Comic edition
+  description: Indicates the specific release version or edition of a comic book, influencing its rarity and collectible value within the market
+  friendly_id: comic_edition
+  handle: comic-edition
+  values:
+  - comic_edition__anniversary_issue
+  - comic_edition__collector's_edition
+  - comic_edition__digital_edition
+  - comic_edition__director's_cut
+  - comic_edition__first_edition
+  - comic_edition__international_edition
+  - comic_edition__reprint
+  - comic_edition__signed_edition
+  - comic_edition__variant_cover
+- id: 3414
+  name: Comic tradition
+  description: The Comic tradition embodies the unique style and themes in comic books
+  friendly_id: comic_tradition
+  handle: comic-tradition
+  values:
+  - comic_tradition__chinese comics
+  - comic_tradition__dutch comics
+  - comic_tradition__franco_belgian_bd
+  - comic_tradition__german_comics
+  - comic_tradition__indian_comics
+  - comic_tradition__indonesian_comics_(maquita)
```

Comment:
We do not support brackets in the handles and friendly ids (it can be omitted).
Here is an example:
![image](https://github.com/user-attachments/assets/0edbf876-0e47-41bc-a24b-95389ca67d96)

Reply:
fixed, thank you